TF_Settings_Web/src/Pages/Camera/Masking/MaskingScreen.tsx

```diff
@@ -170,13 +182,13 @@ const MaskingScreen: React.FC = () => {
                     key={sliderInfo[0]}
                     direction={sliderInfo[0] as SliderDirection}
                     maskingValue={sliderInfo[1]}
-                    canvasInfo={{ size: 800, topOffset: 50, leftOffset: 100 }}
+                    canvasSize={innerHeight < innerWidth ? innerHeight * 0.45 : innerWidth * 0.96 - innerHeight * 0.1}
                     clearMasking={clearMasking}
                     onDrag={setMasking}
                     onDragEnd={sendMaskingRequest}
                 />
             )),
-        [masking.current]
+        [masking.current, innerHeight]
```

Comment:
💬 Do you need to add innerWidth to dependencies as well?

Reply:
Now uses `isLandscape` hook